.dev/compare_branches.R

```diff
@@ -141,22 +144,24 @@ if (is.null(base_branch) || is.na(base_branch) || !nzchar(base_branch)) {
 
 # prioritize "branch"
 is_branch <- FALSE
+has_target <- TRUE
 if (!is.null(params$branch)) {
   branch <- params$branch
   is_branch <- TRUE
 } else if (!is.null(params$pr)) {
   pr <- params$pr
 } else {
-  stop("Please supply a branch (--branch) or a PR number (--pr)")
+  has_target <- FALSE
 }
 
-# prioritize packages
+if (is.null(params$pkg_dir)) {
+  params$pkg_dir <- "."
```

Comment:
Does this default make any sense since it should be run from the lintr project root?

Reply:
Good point... I am not sure this is working as intended at HEAD anyway... just made it an error for now not to provide `pkg_dir` with a TODO & note.

Reply:
By the way: It would be very user-friendly to auto-download packages if `pkg_dir` is missing or doesn't contain any / the requested packages. That's how I initially thought this script worked.

Reply:
Makes sense... added a TODO as an exercise for future readers for now :)